package.json

```diff
@@ -36,6 +36,7 @@
   },
   "homepage": "https://github.com/DW-ReCo/dw-kid2-graph-frontend-prototype#readme",
   "dependencies": {
+    "base-58": "^0.0.1",
```

Comment:
I have added base58, as it is a URL friendly, human readable hashing algorithm to use instead of base64, wheere l and I can be confused as well as O and 0